### ISSUE TITLE
Dynmap

### ISSUE THREAD

**Issue by itsjustnickdev** [gekware/minecraft-server-hibernation-pterodactyl-egg#19]
Any way to add an extra port?
I tried allocating another port for dynmap, but the webserver doesn't start.

**Comment by BolverBlitz** [comment]
Yes you can add extra ports, either when you create the server or the server settings page or you allow the server multiple allocations then your user can add a second port at any time they want in the network tab.

Note your dynmap will only run when the minecraft server isn´t frozen.

**Comment by itsjustnickdev** [comment]
I have tried to add the an extra port and tried starting the mc server, but the dynmap didn't load.
When I did it with the normal paper egg from parker it works fine.

**Comment by BolverBlitz** [comment]
It works the exact same way on this egg.

**Comment by itsjustnickdev** [comment]
Alright will try again in 3 days currently pre-generating chunks.

**Comment by Mazuna** [comment]
I just tested it and Dynmap integration is possible with Pterodactyl and this egg. 
Just configure Dynmap as normal by pasting the mod's .jar into your folder and adjusting the webport inside of dynmap created configuration file. 
Additionally to that you als have to also allocate the port dynmap is going to use to your Minecraft server in your panel.
